packages/nextjs/scripts/buildRollup.ts

```diff
@@ -0,0 +1,24 @@
+import * as childProcess from 'child_process';
+import * as fs from 'fs';
+import * as path from 'path';
+
+/**
+ * Run the given shell command, piping the shell process's `stdin`, `stdout`, and `stderr` to that of the current
+ * process. Returns contents of `stdout`.
+ */
+function run(cmd: string, options?: childProcess.ExecSyncOptions): string | Buffer {
+  return childProcess.execSync(cmd, { stdio: 'inherit', ...options });
+}
+
+run('yarn rollup -c rollup.npm.config.js');
+
+// Regardless of whether nextjs is using the CJS or ESM version of our SDK, we want the code from our templates to be in
+// ESM (since we'll be adding it onto page files which are themselves written in ESM), so copy the ESM versions of the
+// templates over into the CJS build directory. (Building only the ESM version and sticking it in both locations is
+// something which in theory Rollup could do, but it would mean refactoring our Rollup helper functions, which isn't
+// worth it just for this.)
```

Comment:
I think this is the kind of technical debt we will never get rid of. Let's leave it but take the learning that DRY has it's downsides in regards to rollup configs.

Reply:
Interesting - not sure I'd call this tech debt. This is a weird one-off, and I feel like quite often in those cases the answer isn't "make your thing (whatever it is) support the uncommon edge case" it's "find a workaround," which to me is exactly what this is, no?